resources/db/schema/mysql-ddl.sql

```diff
@@ -756,6 +757,17 @@ CREATE TABLE release_comments (
   COLLATE = utf8_unicode_ci
   AUTO_INCREMENT = 1;
 
+  DROP TABLE IF EXISTS release_unique;
+  CREATE TABLE release_unique (
+  releases_id   INT(11) UNSIGNED  NOT NULL COMMENT 'FK to releases.id.',
+  uniqueid BINARY(16)  NOT NULL DEFAULT '0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0' COMMENT 'Unique_ID from mediainfo.',
+  PRIMARY KEY (releases_id),
+  INDEX ix_release_unique_uniqueid(uniqueid)
```

Comment:
Remove the additional index and expand the PRIMARY KEY across releases_id, uniqueid.  In a two column table, you should expand the PK over the two columns if you also intend to add a secondary index over the second column.  This way, both columns are available in a PK lookup and provide additional performance in JOINs with the JOIN on the first column and the WHERE constraint(s) on the second column.  Doing it this way has no detriments as we are not wasting any additional space and only has performance benefits.  This is especially true for MyISAM users as MyISAM does not cluster the PK with the secondary keys.